desk/app/groups.hoon

```diff
@@ -1940,6 +1950,8 @@
     --
   ++  ga-start-join
     ^+  ga-core
+    %-  (log |.("claim: {<cam.gang>}"))
+    =.  cam.gang  `[& %adding]
```

Comment:
This refactoring feels like it should've gone into a separate PR.

Either way, previous logic had `join-all.join` instead of hard-coded `&`. Even if it's always `&` in practice, are we sure we want to hard-code it? Should probably just make this arm a gate, right?